src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,157 @@
+ * Ui class handles interactions with users.
+ */
+public class Ui {
+
+    /**
+     * Constructor of Ui class.
+     */
+    public Ui() {
+
+    }
+
+    public String listTask(TaskList ls) {
+        StringBuilder s = new StringBuilder("Here're the tasks in ur list: \n");
+        int counter = 0;
+        for (int i = 0; i < ls.getSize(); i++) {
+            counter++;
+            Task tk = ls.get(i);
+            s.append(counter+ ". "
+                    + tk.toString()
+                    + "\n");
+        }
+        return s.toString();
+    }
+
+    public String findMatchingTask(String keyWord, TaskList ls) {
+        StringBuilder s = new StringBuilder("Here're the matching tasks in ur list: \n");
+        int counter = 0;
+        for (int i = 0; i < ls.getSize(); i++) {
+            Task tk = ls.get(i);
+            if (tk.getDescription().contains(keyWord)) {
+                counter++;
+                s.append(counter + ". " + tk.toString() + "\n");
+            }
+        }
+        return s.toString();
+    }
+
+
+    /**
+     * Greet users.
+     */
+    public void sayHi() {
+        String hi = "Helloo! I'm LilyBot ;)\nWhat's up'?\n";
+        System.out.print(hi);
+    }
+
+    /**
+     * Say goodbye to users.
+     */
+    public String sayBye() {
+        String s = "Bye Bye. See u later! \n" ;
+        return s;
+    }
+
+    /**
+     * Print to inform users that task is marked done.
+     *
+     * @param task Specifics of the task to be marked done.
+     */
+    public String printMarkDone(String task) {
+        return "Good job! I've marked this task as done: \n"
+                + "  "
+                + task;
+    }
+
+    /**
+     * Print to inform users that task is marked undone.
+     *
+     * @param task Specifics of the task to be marked undone.
+     */
+    public String printMarkNotDone(String task) {
+       return "Okie, Marked this task as not done yet:"
+               + "  "
+               + task;
+    }
+
+    /**
+     * Print to inform users that task is added.
+     *
+     * @param task Specifics of the task to be added.
+     */
+    public String printAdded(String task, TaskList taskList) {
+        return "  Got it. I've added this task:"+ "\n"
+                + "  " + task + "\n"
+                + "  Now u have " + taskList.getSize() +
+                " tasks in the list.";
+    }
+
+    /**
+     * Print to inform users that task is removed.
+     *
+     * @param task Specifics of the task to be removed.
+     */
+    public String printRemoved(String task, TaskList taskList) {
+        return "Noted. Task Removed:" + "\n"
+                + "  " + task + "\n"
+                + "  Now u have " + taskList.getSize() +
+                " tasks in the list.";
+    }
+
+
+
+    /**
+     * Inform users that the description entered is invalid.
+     */
+    public String invalidDescription() {
+        return "Oops! Sorry, I don't know what that means. Description is empty";
+    }
```

Comment:
As this is a method, perhaps you can change the name to a verb? I also noticed the same issue in a few other methods.